Source/Microsoft.Teams.Apps.FAQPlusPlus.Common/Models/AdaptiveSubmitActionData.cs

```diff
@@ -0,0 +1,132 @@
+﻿// <copyright file="AdaptiveSubmitActionData.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace Microsoft.Teams.Apps.FAQPlusPlus.Common.Models
+{
+    using Microsoft.Bot.Schema;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Adaptive Card Action class to post question data.
+    /// </summary>
+    public class AdaptiveSubmitActionData
```

Comment:
This should inherit from TeamsAdaptiveSubmitActionData
Rename this class to something that is less generic.
[Not blocking release]